tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -539,9 +541,10 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
+		[ "$ARCH" == "ppc64le" ] && target="linux-gnu"
```

Comment:
Just to be sure I understand this correctly ... you're forcing "gnu" (glibc) to be used as LIBC for ppc64le, is it?
If so, why? Isn't musl supported on ppc64le world?

I'd make it a explicit fail here, instead of a silent correction, as in:
* check if the arch used is ppc64le
  - check if the LIBC used is gnu
    + If not, raised an error explicitly saying the only expected LIBC type for ppc64le is gnu

If that's the case, a similar patch should be raised on the agent side doing the same.

Reply:
@fidencio musl support is not really great in the ppc64le world. Most of rust toolchain and libraries work for ppc64le is based around glibc.

Reply:
Okay, then I'd take the suggested path and make it a no-go for using musl with ppc64le for now, raising an explicit error.
Does this sound reasonable, @bpradipt?

Reply:
yes, this sounds reasonable @fidencio

Reply:
@bpradipt and @fidencio, I've made the changes. PTAL